gui-tk/minsky.tcl

```diff
@@ -223,6 +224,14 @@ if {$preferences(focusFollowsMouse)} {
 }
 proc setCursor {cur} {. configure -cursor $cur; update idletasks}
 
+# Toggle single or multiple equity column mode
+multipleEquities $preferences(multipleEquities)
```

Comment:
This is the only place your setting Minsky::multipleEquities. You should set it within setPreferenceParms, which is called whenever the preference panel is closed.

Reply:
I only set this in setPreferenceParms now and have reinstated the code in godley.tcl, without actually setting the value there. I just apply it to all Godleys in proc setGodleyDisplay. The feature looks great now. The button widgets appear instantaneously when I deactivate and reactivate the preference. I've also fixed tests t0027a.sh, t0030a.sh, t0033a.sh and t0036a.sh to not save the model upon afterMinskyStarted.